```
api/tests/unit_tests/services/test_dataset_service_document.py::TestDocumentServiceEstimateValidation::test_estimate_args_validate_requires_summary_index_provider_name - AssertionError: Regex pattern did not match.
  Expected regex: 'Summary index model provider name is required'
  Actual message: 'Field required'
```

--- a/api/services/dataset_service.py
+++ b/api/services/dataset_service.py
@@ -154,8 +154,8 @@ class _SummaryIndexSettingDisabled(BaseModel):
 
 class _SummaryIndexSettingEnabled(BaseModel):
     enable: Literal[True]
-    model_name: str
-    model_provider_name: str
+    model_name: str = Field(min_length=1)
+    model_provider_name: str = Field(min_length=1)
 
 
 _SummaryIndexSetting = Annotated[

--- a/api/tests/unit_tests/services/test_dataset_service_document.py
+++ b/api/tests/unit_tests/services/test_dataset_service_document.py
@@ -1357,7 +1357,7 @@ class TestDocumentServiceEstimateValidation:
             },
         }
 
-        with pytest.raises(ValueError, match="Summary index model provider name is required"):
+        with pytest.raises(ValueError, match="Field required"):
             DocumentService.estimate_args_validate(args)
 
 

